src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -71,6 +86,54 @@ object Writer {
     }
   }
 
+  def createBuildTargetFile(buildHeader: String, ts: List[Target], tfp: Path, thirdPartyDirectory: DirectoryName): Result[Int] = {
+    (for {
+          b <- IO.exists(tfp.parent)
+          _ <- if (b) IO.const(false) else IO.mkdirs(tfp.parent)
+          _ <- IO.writeUtf8(tfp, createBuildTargetFileContents(buildHeader, ts, thirdPartyDirectory))
+        } yield ())
+      .map(_ => ts.size)
```

Comment:
this is weird, can we not just do `yield ts.size` rather than adding the map on?

Reply:
👍